tests/test_optimizer/_utils.py

```diff
@@ -16,6 +16,11 @@
 )
 
 
+def print_rank_0(msg):
+    if dist.get_rank() == 0:
+        print(msg)
```

Comment:
This function is redundant. Use dist coordinator

Reply:
I think a helper for testing is fine?